service/svc.sh

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env bash
+
+ROOTPROJECTPATH="$(
+    cd -- "$(dirname "$0")/.." >/dev/null 2>&1
+    pwd -P
+)"
+
+function install {
+    path=$1
+    if [ -z "$1" ]; then
+        path="/lib/systemd/system"
+    fi
+
+    if [ -e "$path/threeal-bot.service" ]; then
+        echo "Service is already installed"
+        exit
+    fi
+
+    go_path=$(which go)
+
+    workdir=$ROOTPROJECTPATH
+
+    user=$(logname)
+
+    echo "Installing service..."
+
+    sed -e "s@<user>@$user@g" -e "s@<workdir>@$workdir@g" -e "s@<goabsolutepath>@$go_path@g" service/threeal-bot.service >threeal-bot.service
+
+    mv threeal-bot.service $path
```

Comment:
ooh right, since we're going to use `mv` command, first let's redirect to `/tmp` directory instead.
```bash
... > /tmp/threeal-bot.service
mv /tmp/threeal-bot.service $path
```